moment/moment-node.d.ts

```diff
@@ -374,7 +374,22 @@ declare module moment {
     interface MomentStatic {
         version: string;
         fn: Moment;
-
+        
+        default(): Moment;
+        default(date: number): Moment;
+        default(date: number[]): Moment;
+        default(date: string, format?: string, strict?: boolean): Moment;
+        default(date: string, format?: string, language?: string, strict?: boolean): Moment;
+        default(date: string, formats: string[], strict?: boolean): Moment;
+        default(date: string, formats: string[], language?: string, strict?: boolean): Moment;
+        default(date: string, specialFormat: () => void, strict?: boolean): Moment;
+        default(date: string, specialFormat: () => void, language?: string, strict?: boolean): Moment;
+        default(date: string, formatsIncludingSpecial: any[], strict?: boolean): Moment;
+        default(date: string, formatsIncludingSpecial: any[], language?: string, strict?: boolean): Moment;
+        default(date: Date): Moment;
+        default(date: Moment): Moment;
+        default(date: Object): Moment;
```

Comment:
where does this method come from? I don't see it in the moment docs. also there are no tests for this